packages/react-dom/src/__tests__/ReactDOMFizzShellHydration-test.js

```diff
@@ -245,9 +277,9 @@ describe('ReactDOMFizzShellHydration', () => {
       root.render(<Text text="New screen" />);
     });
     expect(Scheduler).toHaveYielded([
+      'New screen',
```

Comment:
These switched order because the recoverable error gets logged during the commit phase instead of immediately in the event handler.